helm/cluster-azure/templates/helmreleases/_helpers.tpl

```diff
@@ -0,0 +1,9 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Common Values for helm releases
+*/}}
+{{- define "values.common" -}}
+provider: capz
+{{- end -}}
+
```

Comment:
Is this really necessary? It's only used once, so I'd inline `provider: capz` where is needed.

Reply:
I thought there might be more going forward but sure :+1: